gulpfile.js

```diff
@@ -403,7 +403,8 @@ function thirdPartyBootstrap(watch, shouldMinify) {
   console.log('Processing ' + input);
   var html = fs.readFileSync(input, 'utf8');
   var min = html;
-  min = min.replace(/\.\/integration\.js/g, './f.js');
+  min = min.replace(/\.\/integration\.js/g,
```

Comment:
This doesn't look right. Does it? You replacing a part of URL with the absolute URL?